src/lib/autoinstall/clients/inst_autosetup_upgrade.rb

```diff
@@ -226,6 +226,23 @@ def main
           false
         )
 
+        #
+        # Checking if at least one base product has been selected/evaluated.
+        #
+        begin
+          Product.FindBaseProducts
+        rescue StandardError
```

Comment:
`FindBaseProducts` throws `RuntimeError` exception [here](https://github.com/yast/yast-yast2/blob/master/library/packages/src/modules/Product.rb#L129) (when the exception class is not specified then `RuntimeError` is raised).

Reply:
RuntimeError is the direct descendant of StandardError. 
https://airbrake.io/blog/ruby-exception-handling/runtimeerror
I have tested it. It is working :-)